active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,655 @@
+even if the element is blocked by another element,
+even if the element is transparent.
+
+This method is typically used with the Window's `ExtendContentIntoTitleBar` property 
+set to true in order to hide the default system title bar.
+However, even when the default system title bar is not hidden, SetTitleBar can be used to make 
+additional regions in your app behave like the title bar.
+
+The following shows a Window with a no built-in title bar. 
+Instead it uses the whole window as its content area. To enable features such as window dragging, 
+it designates a TextBlock as the title bar.
+
+```xml
+<Window ...>
+    <Grid>
+        <!-- ... -->
+
+        <TextBlock x:Name="CustomTitleBar">Custom title text</TextBlock>
+
+        <!-- ... -->
+    </Grid>
+
+</Window>
+```
+
+```CS
+private Window m_window;
+
+protected override void OnLaunched(Microsoft.UI.Xaml.XamlLaunchActivatedEventArgs args)
+{
+    m_window = new MainWindow();
+
+    m_window.ExtendContentIntoTitleBar = true;
+    m_window.SetTitleBar(m_window.CustomTitleBar);
+
+    m_window.Activate();
+}
+```
+
+Title bars should be have background and foreground colors based on
+whether the window is current foreground (active) or background. Additionally, there's an 
+option in Windows Settings to use the user's accent color as the foreground of the title bar.
+See [link TBD] for a sample application that implements this.
+
+
+## Window.Title property
+
+Get or set a string to display as the window's title.
+
+This title is displayed in the windows title bar (unless `ExtendsContentIntoTitleBar` is set),
+as well as other locations such as when the user presses Alt-Tab to bring up the application switcher.
+
+### Example
+
+```XML
+<Window 
+    x:Class="MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    Title="Main Window" >
+    
+</Window>
+```
+
+### Remarks
+
+In a UWP app, this property is a wrapper for 
+[ApplicationView.Title](https://docs.microsoft.com/uwp/api/Windows.UI.ViewManagement.ApplicationView.Title). 
+In a Desktop app this is a wrapper for 
+[SetWindowText](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setwindowtexta).
+
+## Window.Current property
+_Behavior update to existing API when running as Desktop app_
+
+Gets the window of the current thread.
+
+### Remarks
+
+The value of this property depends on the thread from which it is called. 
+If called from a UI thread of a UWP app, the value is the Window instance for that thread. 
+On any other thread, the value is null.
+
+## Window.CoreWindow property
+_Behavior update to existing API when running as Desktop app_
+
+Gets the [CoreWindow](https://docs.microsoft.com/uwp/api/Windows.UI.Core.CoreWindow) associated 
+with this Window. In a Desktop app this is always null.
+
+```CS
+public CoreWindow CoreWindow { get; }
```

Comment:
There's no reason to expose CoreWindow and CoreDispatcher. If UWP apps really need these, just use the existing static methods on those types. (i.e. CoreWindow.GetForCurrentThread)